### ISSUE TITLE
`Unexpected token ILLEGAL` when using ES6 template strings in JSX props

### ISSUE THREAD

**Issue by lencioni** [eslint/eslint#2045]
With the following file:

```jsx
const React = require('react/addons');
const Test = require('test');

var namePart = 'testing';
var test = <Test name={`name${namePart}`} />;

module.exports = test;
```

I get the following output:

> tmp.jsx
>   8:1  error  Unexpected token ILLEGAL
> 
> ✖ 1 problem (1 error, 0 warnings)

ESLint 0.17.0.

Heres a bit of my configuration file:

```yaml
  env:
    browser: true
    es6: true
    node: true

  ecmaFeatures:
    jsx: true
```

Am I doing something wrong here? Let me know if you need any more information.

**Comment by lencioni** [comment]
Sometimes I also see messages like:

>   182  -1  error  Unexpected quasi
>        myProp={

In cases where I am using ES6 template strings in props as well.

**Comment by ilyavolodin** [comment]
Looks like a bug in Espree.

**Comment by lencioni** [comment]
Yep, looks like it has been reported over there too. https://github.com/eslint/espree/issues/95

**Comment by LongLiveCHIEF** [comment]
Anyone else running into this again with the current latest version of eslint/espree? (eslint@0.20.0, espree@2.0.2)

**Comment by nzakas** [comment]
Can you please open a new issue? Closed issues don't get much attention and it's very hard to know if it's the same issue or a different one.

**Comment by LongLiveCHIEF** [comment]
I'm going to do a dev install and run the tests first to make sure it's not a PEBKAC problem.
